watertap/data/techno_economic/media_filtration.yaml

```diff
@@ -2,25 +2,18 @@ default:
   energy_electric_flow_vol_inlet:
     value: 0.00015 #todo: revisit this value later on
     units: kWh/m^3
-  specific_capital_cost_flow_vol_inlet:
-    flow_basis_par:
+  capital_cost:
```

Comment:
This model has a different form in the old WT3 files, so it will need custom code (unless something has changed).

That said, we should check with @kurbansitterley where these values came from seeing as they were already here.

Reply:
OK. The previous yaml file does not make sense, I will build the custom costing model for this media filtration unit.

Reply:
@kurbansitterley The issue is that we don't know which of the two version we should be using  -the basic unit version or the one in WT3's `media_filtration.py`.

Reply:
They are both used throughout the case studies with very similar results. I don't know why that was done. I don't think there is any use in porting both of them. I linked to the `media_filtration.py` WT3 model in our tracking spreadsheet and marked it as not a basic unit because in general I think it is better to have cost based on some physical/operational parameter (in this case e.g. loading rate and filter surface area) rather than just flow (as is the case with the basic_units). 

Using the `media_filtration.py` version rather than the basic_unit version will result in slightly different TEA results, but as we've discussed, we are likely going to have different TEA results in the watertap version vs. those from WT3 for many reasons so I don't think that is a reason to bring one over the other.